packages/cli/src/commands/setup/auth/providers/netlify.js

```diff
@@ -1,7 +1,10 @@
 // the lines that need to be added to index.js
 export const config = {
-  imports: [`import netlifyIdentity from 'netlify-identity-widget'`],
-  init: 'netlifyIdentity.init()',
+  imports: [
+    `import netlifyIdentity from 'netlify-identity-widget'`,
+    `import { isBrowser } from '@redwoodjs/prerender/browserUtils'`,
+  ],
+  init: 'if (isBrowser) {\n  netlifyIdentity.init()\n}',
```

Comment:
Ah, I didn't even consider adding this at the time of setup. Was thinking about including the instructions in the documentation.

Suggestion: add a comment above the line stating this is a check needed for Prerender. Fine as if whether or not pretender is being used.

Reply:
I tried, but really doesn't like comments :/. Errors out